tests/acceptance.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
```

Comment:
Could you please add short description about the script?

sample:
```
# acceptance test script for each command line options.
```

Reply:
sure